Classes/Driver/AmazonS3Driver.php

```diff
@@ -1139,7 +1180,7 @@ public function getFoldersInFolder(
         array $folderNameFilterCallbacks = [],
         $sort = '',
         $sortRev = false
-    ) {
+    ): array {
         if ($start === false && $numberOfItems === false) {
```

Comment:
In hoeverre moeten we (nu al) iets met strict typing qua functie params? We definieren `$start = 0` en controleren vervolgens of ie niet `false` is. Beetje lelijk dit zo

Reply:
Dit gaat niet vanwege de `AbstractDriverInterface` waar return types ontbreken